src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,29 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    private List<Bulldozer> bulldozers = new ArrayList<>();
+
+    @Override
+    public List<Bulldozer> get() {
```

Comment:
Consider creating a local variable in your implementation of `get()` method. It's better to create a new list inside the method and return it, rather than using a class field which could be accidentally modified from outside of this method.